specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/examples/entities/GetQueries.json

```diff
@@ -0,0 +1,457 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "operationalInsightsResourceProvider": "Microsoft.OperationalIinsights",
+    "entityId": "e1d3d618-e11f-478b-98e3-bb381539a8e1",
+    "kind": "Insight"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspaces/myWorkspace/providers/Microsoft.SecurityInsights/queries/6db7f5d1-f41e-46c2-b935-230b36a569e6",
```

Comment:
shouldn't it be "entities/{entity-id}/queries"??
you have only "queries"

Reply:
Let's talk. I think that this API should not have arm id and type at all (I didn't wrote this API, I only "Swagger" it.)

Reply:
Fixed